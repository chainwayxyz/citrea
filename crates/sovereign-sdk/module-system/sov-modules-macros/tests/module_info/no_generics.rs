use sov_modules_api::ModuleInfo;

#[derive(ModuleInfo)]
struct TestStruct {}

fn main() {}
