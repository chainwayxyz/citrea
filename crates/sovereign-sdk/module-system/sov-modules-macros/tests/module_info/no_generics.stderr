error: No generic parameters found
 --> tests/module_info/no_generics.rs:3:10
  |
3 | #[derive(ModuleInfo)]
  |          ^^^^^^^^^^
  |
  = note: this error originates in the derive macro `ModuleInfo` (in Nightly builds, run with -Z macro-backtrace for more info)
